include/schema/common/footer.json

```diff
@@ -4,16 +4,6 @@
     "description": "Page footer configurations",
     "type": "object",
     "properties": {
-        "copyright": {
-            "$ref": "/misc/copyright.json",
-            "description": "Copyright text",
```

Comment:
No, don't remove the schema. We still need this section to verify user's configuration file. What I mean is just removing `"$ref": "/misc/copyright.json"`